src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  health = 100;
+  static alive = [];
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  static get alive() {
+    return this.alive;
+  }
+  static set alive(aliveAnimals) {
+    this.alive = aliveAnimals;
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  hidden = false;
+
+  hide() {
+    this.hidden = true;
+  }
 }
 
 class Carnivore extends Animal {
-  // write your code here
+  bite(target) {
+    if (target.hidden === false) {
```

Comment:
```suggestion
    if (!target.hidden) {
```
and the bite method should check if the target is an instance of Herbivore, fix it please

Reply:
Sure, it's not always better to have less code, better to make it more readable and predictable instead.